### DISCUSSION TITLE
What's the current link to the nvim-spectre example?

### DISCUSSION THREAD

**Discussion by mrjones2014** [grapp-dev/nui-components.nvim#10] [Q&A] [answered]
In the showcase (https://nui-components.grapp.dev/docs/showcase) the link to the source code in the linked Reddit post is a dead link.

Can you provide the source code for that example?

**Comment by mobily** [comment, marked as answer]
@mrjones2014 sure! here's the link: https://github.com/mobily/.nvim/blob/main/lua/pickers/spectre/init.lua
